react/ProductList.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { FunctionComponent } from 'react'
+import { FormattedMessage } from 'react-intl'
 
 import ListItem from './components/ListItem'
+import Availability from './constants/Availability'
```

Comment:
⛏ I would import `AVAILABLE` directly, I feel it's easier to read.

Reply:
Fair point.